src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -80,10 +80,10 @@ public function drop(ConnectionInterface $db)
  * In all other cases where the initial and current hash differs, assume
  * the table has changed
  *
- * @param DboSource $db
+ * @param ConnectionInterface $db
  * @return boolean
  */
-    protected function _tableUnmodified($db)
+    protected function _tableUnmodified(ConnectionInterface $db)
```

Comment:
Adding typehint strictly speaking would be non BC, so let's skip it. We can add it for new major Cake 4 compatible version of the plugin.

Reply:
Well, all other methods in this class have type hints including the method that calls `_tableUnmodified`.